### PR TITLE
Added v2 schema with profile

### PR DESCRIPTION
**Pull request by mikluke**
Changes:

* PDV object's schema is changed
* Cookie's schema is changed
* Profile Type added

### CODE REVIEW

File: internal/schema/v1/profile.go [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+	"github.com/Decentr-net/cerberus/internal/schema/types"
+)
+
+const (
+	maxFirstNameLength = 64
+	maxLastNameLength  = 64
+)
+
+// Profile is PDVData implementation for profile's data.
+type Profile struct {
```
Comment (megaherz) [inline internal/schema/v1/profile.go:15]: Why avatar is omitted?

Comment (mikluke) [reply internal/schema/v1/profile.go:18]: returned

File: internal/storage/index_storage.go [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+import (
+	"context"
+	"time"
+)
+
+//go:generate mockgen -destination=./mock/index_storage.go -package=mock -source=index_storage.go
+
+// IndexStorage provides access to pdv index.
+type IndexStorage interface {
```
Comment (megaherz) [inline internal/storage/index_storage.go:11]: Why index storage?

Comment (mikluke) [reply internal/storage/index_storage.go:11]: because it's an index for pdv

**Pull request merged**